pymemcache/client/base.py

```diff
@@ -304,6 +305,9 @@ def set(self, key, value, expire=0, noreply=None):
                   from the cache, or zero for no expiry (the default).
           noreply: optional bool, True to not wait for the reply (defaults to
                    self.default_noreply).
+          flag: remcached operation flag
+          encoding: pymemcached use ascii, remcached use utf-8
```

Comment:
Again there is no need to reference remcached here as this is a generic, public-facing API. The encoding value here controls how the data is encoded before shipping it on the wire. The fact that remcached chooses `utf-8` is an implementation detail.